src/SDK/Trace/Span.php

```diff
@@ -21,6 +21,86 @@
 
 final class Span extends API\AbstractSpan implements ReadWriteSpanInterface
 {
+
```

Comment:
is this a formatting/EOL change? The code doesn't look different

Reply:
Don't really know. It's LF line breaks and I don't think it was different before.  That's probably something the CS fixer did, I just changed StatusData to StatusDataInterface in the file.